nuxt.config.js

```diff
@@ -129,6 +131,13 @@ export default {
         id: process.env.GOOGLE_TAG_MANAGER_ID,
     },
 
+    facebook: {
+        track: 'PageView',
+        pixelId: '561487855431220',
+        autoPageView: true,
```

Comment:
**question:** is the Pixel ID value should be imported from the `process.env` like the setting of GOOGLE_TAG_MANAGER_ID? It could avoid recording the behaviors made when developing if we don't set the value.

Reply:
this module handles the situation and does not record anything if the app is launched with `npm run dev`.

Reply:
Oh, I got it. Thanks for the explanation.